### DISCUSSION TITLE
The rdp service is not working.

### DISCUSSION THREAD

**Discussion by Abygail007** [amidaware/tacticalrmm#1774] [Q&A]
Hello everybody!

I have a little problem. I have a TacticalRmm installation on a Linux server. All my Windows 10 machines are integrated into it (280). Everything works great! But there is just the "RDP" connection which fails to work. Nor by the “meshcentral” either. I tested without our antivirus system but same problems.

Have you had this problem too?

Thanks for the help!

**Comment by dinger1986** [comment]
Hello, do you mean at the top?

![image](https://github.com/amidaware/tacticalrmm/assets/7244447/c240f44c-7ffc-44bf-bdec-6fe6788c3ea4)

If so thats still using meshcentral so if it doesnt work there it wont work in tactical, it probably needs the config.json file adjusted, or a setting on the machine you are trying to RDP to maybe this helps? https://www.reddit.com/r/MeshCentral/comments/14g8a41/rdp_stuck_on_setup/

**Reply by Abygail007** [reply]
Hello, yes this option.
I see on the meshcentral forum, if they see where this comes from. THANKS

**Reply by silversword411** [reply]
Are you trying to connect to a win machine that has an RDP capabilities? Pro or server. Not Home
Have you manually enabled RDP on the machine you're trying to connect to?
I haven't tested too much, but there might be a possibility you need to check firewall rules as well.

All that option does is connect using a web based RDP client to the native RDP of windows.

**Reply by silversword411** [reply]
https://docs.tacticalrmm.com/mesh_integration/#take-control-connect-vs-rdp-connect

**Reply by Abygail007** [reply]
Hello, thank you for your help. Yes, I only did this on Windows 10 Pro and on Windows Server. RDP has been enabled on these machines.
On the firewall, I checked on my physical firewall everything is OK. I uninstalled the antivirus (which is the system firewall) and deactivated the Windows firewall, but it didn't work.

**Reply by silversword411** [reply]
I'd have to login and check. I don't use it often, but the RDP has worked fine for me when I've tested in the past. I can have a quick look if you open a ticket and give ssh access.
https://docs.tacticalrmm.com/support/#commercial-support
https://docs.tacticalrmm.com/support/#public-rsa-keys

**Reply by alencarwell** [reply]
> Eu teria que fazer login e verificar. Não o uso com frequência, mas o RDP funcionou bem para mim quando testei no passado. Posso dar uma olhada rápida se você abrir um tíquete e der acesso ssh. [https://docs.tacticalrmm.com/support/#commercial-support ](https://docs.tacticalrmm.com/support/#commercial-support) https://docs.tacticalrmm.com/support/#public-rsa-keys

hello, i have some problem, do you helpme?

**Comment by alencarwell** [comment]
hello, i have some problem, do you helpme?
